projects/target-platforms/packages/create-plaform/bin/create-platform.js

```diff
@@ -82,6 +82,9 @@ const output = {
 		sources: ['assets', 'src'],
 	},
 	pkgJson: {
+		bin: {
+			'liferay-platform-scripts': './liferay-platform-scripts.js',
```

Comment:
I'm a little hesitant with the naming since we already have `liferay-workspace-scripts` and `liferay-npm-scripts`. Adding another `liferay-*-scripts` might just add some confusion later down the line.

Would doing something like `liferay-platform` make more sense and reduce the chance of confusion? I can't think of a great name at the moment, but I will think about this more... Ideally customers would only have to set this once and whenever they go from 7.4-GA4 to 7.5-GA1, they would just update their dependency.

Reply:
I had the same problem and thought about `liferay-platform` too :sweat_smile:, but seemed a bit awkward: 

`liferay-platform build`

Looks like you are building the platform, not *for* the platform...

Reply:
In any case, we can merge it, and change it to a better name in the future.

Reply:
Maybe `liferay` alone? :thinking: 
So that command would be `liferay build`. 

That way you would even be able to do things like: `yarn liferay lint` or `yarn liferay deploy` without having to create anything in the `scripts` section of the `package.json`...

Reply:
Did a quick industry-wide poll:

| Product | npm cli package | cli binary |
| --- | --- | --- |
| [Netlify](https://docs.netlify.com/cli/get-started/) | `netlify-cli` | `netlify` |
| [Salsesforce](https://developer.salesforce.com/docs/atlas.en-us.232.0.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm) | `sfdx-cli` | `sfdx` |
| [Next](https://nextjs.org/docs/api-reference/cli) | `next` | `next` |
| [Vercel](https://vercel.com/cli) | `vercel` | `vercel` |
| [Shopify](https://shopify.dev/tools/cli/installation) | `shopify-cli` | `shopify` |

Based on this, I think ideally we might want to have some workflow like:

```bash
> npm i -g liferay-cli
> liferay build
```

Reply:
Yeah we aren't really a `*-cli` package since this is used primarily via an npm script. I could imagine a cli package in the future for generating themes or portlets like @izaera mentioned. So maybe we also want to think about that in relation to this. I do like the idea of saving the `liferay` bin for something you globally install. What about `liferay-target`?

| npm cli package | cli binary |
| --- | --- |
| `@liferay/cli` (package TBD) | `liferay new theme` |
| `@liferay/target-7.4-GA1` | `liferay-target build` |
| `@liferay/target-7.3-GA4` | `liferay-target build` |

Reply:
We have a naming scheme for the platform packages already, @nhpatt gave us the [pattern to follow](https://github.com/liferay/liferay-frontend-projects/issues/544#issuecomment-842991531), based on how the Java platforms are named.

Regarding the `liferay` binary, my proposal was to:

1. Embed the `liferay` script in the platforms for now. That would allow us to put `"build": "liferay build"` in the `package.json`.
2. When we create the tool, give it a package name (eg: `@liferay/cli`) and make it install a binary called `liferay` too. This way uses can do `liferay new theme` for example.
3. Make `@liferay/cli`'s `liferay` use the local `liferay` in projects that have it, as other standard tools (eg: `lerna`, and I believe `ng`) do, which let you install them globally and locally and the global one delegates to the local one as needed.

This would let us maintain backwards compatibility without the need to release `@liferay/cli` now or make breaking changes in the platforms in the future. I think... :thinking:

Reply:
I like that idea and I think you are right that by creating that binary now we don't actually have to worry about when we make the new package. So I am all for using just `liferay`

Reply:
I've pushed more commits to account for the renaming.